src/packages/fs/utils/is-js-file.js

```diff
@@ -0,0 +1,4 @@
+// filter out hidden files && non .js files
+export default function isJsFile(file) {
```

Comment:
Can we make this function named `isJSFile` for consistency with traditional capitalization for the JS abbreviation?